src/component/DefaultTooltipContent.tsx

```diff
@@ -79,7 +79,7 @@ export class DefaultTooltipContent<TValue extends ValueType, TName extends NameT
         };
         const finalFormatter = entry.formatter || formatter || defaultFormatter;
         let { value, name } = entry;
-        if (finalFormatter && value && name) {
+        if (finalFormatter ?? value ?? name) {
```

Comment:
I believe this should check if the value and name are not null or undefined? With `??` value and name can be undefined, which I don't think is the intention here.

```suggestion
        if (finalFormatter && value != null && name != null) {
```

Reply:
You're right! I'm still getting used to the `??` operator. Committing this change